packages/datagateway-dataview/src/views/table/datasetTable.component.test.tsx

```diff
@@ -253,6 +254,7 @@ describe('Dataset table component', () => {
           parentEntities: [],
         },
       ],
+      isLoading: false,
```

Comment:
There's another `useCart` mock function that needs `isLoading: false`, on line 274 below this